setup/install_gatling/30_classification/40_classify_nodes_via_nc.rb

```diff
@@ -3,4 +3,12 @@
 test_name "Classify PE agents via Node Classifier" do
   skip_test "Installing FOSS, not PE" unless ENV["BEAKER_INSTALL_TYPE"] == "pe"
   classify_nodes_via_nc
+
+  # create non-perf-agent-group and use it to add ntp
+  classifier.find_or_create_node_group_model(
+    "parent"  => "00000000-0000-4000-8000-000000000000",
+    "name"    => "non-perf-agent-group",
+    "rule"    => ["and", ["not", ["~", %w[fact clientcert], ".*agent.*"]]],
```

Comment:
any reason not to go with ".*perf-agent.*"?

Reply:
Since it is the inverse of the "perf-agent-group" I chose to use the exact regex used in the [classify_nodes_via_nc](https://github.com/puppetlabs/gatling-puppet-load-test/blob/master/setup/helpers/perf_helper.rb#L391-L400) helper that creates that group.

Reply:
That is a good reason. :)